pkg/virt-config/virt-config.go

```diff
@@ -236,7 +235,7 @@ func (c *ClusterConfig) GetOVMFPath(arch string) string {
 	case "ppc64le":
 		return c.GetConfig().ArchitectureConfiguration.Ppc64le.OVMFPath
 	case "s390x":
-		return ""
+		return c.GetConfig().ArchitectureConfiguration.S390x.OVMFPath
```

Comment:
Would it make sense to just ignore the value here, or skip the field in the s390x api entirely. I know the last one would require a new custom type, but code wise it should not be to different.